pkg/aws/client.go

```diff
@@ -168,7 +169,17 @@ func (b *ClientBuilder) Build() (result Client, err error) {
 		awsSession:          sess,
 	}
 
-	return
+	_, root, err := getClientDetails(c)
+	if err != nil {
+		err = fmt.Errorf("failed to get client details: %v", err)
+		return nil, err
+	}
+
+	if root {
+		return nil, errors.New("root account shouldn't be used, please use IAM account")
```

Comment:
Can we make that sound a little more friendly? end of the day that's a customer hitting something unexpected at the very beginning of their customer journey

Reply:
@RiRa12621  Updated, PTAL, thanks!